upi/openstack/03_bootstrap.yaml

```diff
@@ -22,7 +25,6 @@
       allowed_address_pairs:
       - ip_address: "{{ os_subnet_range | next_nth_usable(5) }}"
       - ip_address: "{{ os_subnet_range | next_nth_usable(6) }}"
-      - ip_address: "{{ os_subnet_range | next_nth_usable(7) }}"
```

Comment:
Thanks for the heads up, this has been extracted to https://github.com/openshift/installer/pull/2848